style/amenity-points.mss

```diff
@@ -2286,6 +2287,9 @@
       [feature = 'amenity_college'],
       [feature = 'amenity_university'] {
         text-fill: darken(@societal_amenities, 80%);
+      }
+	  [feature = 'amenity_research_institute'] {
```

Comment:
You can merge this entry with the block of kindergarten, school, college and university.

Reply:
Emmm……but kindergarten or school are separated, so I think institute should be like that?

Reply:
Are they?
![image](https://user-images.githubusercontent.com/2410353/104010238-7be8a100-51ac-11eb-8329-a59b5cbe5b71.png)

Reply:
Yes
I‘m new in this repo.
I guess your expected code is like that？
```
      [feature = 'amenity_kindergarten'],
      [feature = 'amenity_school'],
      [feature = 'amenity_college'],
      [feature = 'amenity_university' || feature = 'amenity_research_institute'] {
        text-fill: darken(@societal_amenities, 80%);
      }
```
（I don't know whether those is Grammatical (T_T)）

Reply:
> I guess your expected code is like that？

No:
```
[feature = 'amenity_school'],
      [feature = 'amenity_college'],
      [feature = 'amenity_university'],
      [feature = 'amenity_research_institute'] {
```
Cartocss is based on CSS syntax.

Reply:
Edited.Thanks @HolgerJeromin